lib/resin-lint.ts

```diff
@@ -146,7 +145,11 @@ const lintTsFiles = async function(
 				return 1;
 			}
 		}
-		linter.lint(file, source, config as tslint.Configuration.IConfigurationFile);
+		linter.lint(
+			file,
+			source,
+			config as tslint.Configuration.IConfigurationFile,
```

Comment:
Would it be better to run `linter.lint()` _before_ `prettier.format()`? Because, with the `--fix` option, `linter.lint()` could modify the source code (right?), in which case a subsequent `prettier.format()` call could further modify the source code. In other words, it looks like running `resin-lint --typescript --fix` could result in an "unstable" state where running `resin-lint --typescript --fix` a second time would result in further source code changes.

_(Comment copied from https://github.com/balena-io/node-resin-lint/pull/44/files#r374730796)_

Reply:
It does seem like it would be possible (although unlikely due to the nature of the fixes tslint can apply). I'll have to dig into whether tslint's autofix also checks for and/or applies prettier fixes and see whether I can merge/move around this code. It'll be a separate PR though as this one doesn't actually change the autofix functionality at all

Reply:
https://github.com/balena-io/node-resin-lint/pull/46